hpx/runtime/components/colocating_distribution_policy.hpp

```diff
@@ -181,34 +181,47 @@ namespace hpx { namespace components
         /// \note This function is part of the invocation policy implemented by
         ///       this class
         ///
+        template <typename Action, typename Continuation, typename ...Ts>
+        bool apply(Continuation && c,
+            threads::thread_priority priority, Ts&&... vs) const
+        {
+            if (!id_)
+            {
+                return hpx::detail::apply_impl<Action>(std::forward<Continuation>(c),
+                    hpx::find_here(), priority, std::forward<Ts>(vs)...);
+            }
+            return hpx::detail::apply_colocated<Action>(
+                std::forward<Continuation>(c), id_, std::forward<Ts>(vs)...);
+        }
+
```

Comment:
Let's have it for symmetry reasons anyways. Later on somebody might be puzzled why this isn't there otherwise.